src/Program.cs

```diff
@@ -27,16 +30,19 @@ public static void Main(string[] args)
                         opts.Thumbprint,
                         Enum.Parse<StoreName>(
                             opts.StoreName,
+                            ignoreCase: true),
+                        Enum.Parse<StoreLocation>(
+                            opts.StoreLocation,
                             ignoreCase: true)))
                 .WithNotParsed(
                     errs =>
                         Console.WriteLine(
                             $"Error parsing\n {string.Join('\n', errs)}"));
         }
 
-        private static void RemoveCertificate(string path, string base64, string password, string thumbprint, StoreName storeName)
+        private static void RemoveCertificate(string path, string base64, string password, string thumbprint, StoreName storeName, StoreLocation storeLocation)
         {
-            Console.WriteLine($"Removing certificate from '{path}' from current user's '{storeName}' certificate store...");
+            Console.WriteLine($"Removing certificate from '{path}' from '{storeName}' certificate store (location: {storeLocation})...");
```

Comment:
can you move this line into the `if(!string.IsNullOrEmpty(path))` underneath please?